### ISSUE TITLE
Update.

### ISSUE THREAD

**Issue by gman4** [amnezia-vpn/amnezia-client#1253]
Annoying bug in google play version 4.8.2.4 and here is 4.8.2.3 and it cant be updated.

**Comment by Korb** [comment]
Indeed, Google Play currently offers version 4.8.2.4 (Nov 8, 2024) for installation. To which version are you currently unable to update your application installed via Google Play?
